[2025-08-18T08:05:41.831+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-08-18T08:05:41.920+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: youtube_pipeline_exec.ingest_raw manual__2025-08-18T08:05:41.278820+00:00 [queued]>
[2025-08-18T08:05:41.936+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: youtube_pipeline_exec.ingest_raw manual__2025-08-18T08:05:41.278820+00:00 [queued]>
[2025-08-18T08:05:41.937+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 2
[2025-08-18T08:05:41.971+0000] {taskinstance.py:2327} INFO - Executing <Task(BashOperator): ingest_raw> on 2025-08-18 08:05:41.278820+00:00
[2025-08-18T08:05:41.986+0000] {standard_task_runner.py:63} INFO - Started process 256 to run task
[2025-08-18T08:05:42.003+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'youtube_pipeline_exec', 'ingest_raw', 'manual__2025-08-18T08:05:41.278820+00:00', '--job-id', '30', '--raw', '--subdir', 'DAGS_FOLDER/youtube_pipeline.py', '--cfg-path', '/tmp/tmp8o_7u1ru']
[2025-08-18T08:05:42.008+0000] {standard_task_runner.py:91} INFO - Job 30: Subtask ingest_raw
[2025-08-18T08:05:42.147+0000] {task_command.py:426} INFO - Running <TaskInstance: youtube_pipeline_exec.ingest_raw manual__2025-08-18T08:05:41.278820+00:00 [running]> on host 545418c0948b
[2025-08-18T08:05:42.476+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='youtube_pipeline_exec' AIRFLOW_CTX_TASK_ID='ingest_raw' AIRFLOW_CTX_EXECUTION_DATE='2025-08-18T08:05:41.278820+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-08-18T08:05:41.278820+00:00'
[2025-08-18T08:05:42.477+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-08-18T08:05:42.508+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-08-18T08:05:42.509+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'docker exec youtube-pipeline-project_spark-master_1 spark-submit --packages org.apache.spark:spark-avro_2.12:3.5.1 /app/batch/ingestion/ingest_raw_to_hdfs.py']
[2025-08-18T08:05:42.523+0000] {subprocess.py:86} INFO - Output:
[2025-08-18T08:05:42.551+0000] {subprocess.py:93} INFO - Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
[2025-08-18T08:05:42.552+0000] {subprocess.py:97} INFO - Command exited with return code 1
[2025-08-18T08:05:42.553+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-08-18T08:05:42.573+0000] {taskinstance.py:2890} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/operators/bash.py", line 243, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2025-08-18T08:05:42.583+0000] {taskinstance.py:1205} INFO - Marking task as UP_FOR_RETRY. dag_id=youtube_pipeline_exec, task_id=ingest_raw, execution_date=20250818T080541, start_date=20250818T080541, end_date=20250818T080542
[2025-08-18T08:05:42.604+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 30 for task ingest_raw (Bash command failed. The command returned a non-zero exit code 1.; 256)
[2025-08-18T08:05:42.633+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2025-08-18T08:05:42.693+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-08-18T08:05:42.695+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
